src/tools/deploy-test-contract.ts

```diff
@@ -21,8 +25,13 @@ const TOKENS_TO_CREATE = 10000000000000; // ten trillion tokens
   LoggerFactory.INST.logLevel("error");
 
   // ~~ Initialize SmartWeave ~~
-  const smartweave = WarpNodeFactory.memCached(arweave);
-
+  const warp = WarpFactory.forMainnet(
+    {
+      ...defaultCacheOptions,
+      inMemory: true,
+    },
+    true
+  );
```

Comment:
@vilenarios - should we use `forTestnet()` on these?

Reply:
Yeah definitely!